build.gradle.kts

```diff
@@ -138,7 +138,9 @@ tasks.validateBeforeBuildingReleaseArtifacts {
 
 val String.v: String get() = rootProject.extra["$this.version"] as String
 
+// Looker Instructions: Keep the first line with the suffix for development snapshots; use the second for release builds.
```

Comment:
what do you think of adding a flag to the invocation of the script `--snapshot` or something and if that flag is used it changes this accordingly? similar to how we update gradle.properties in looker-release.sh.

remembering to comment or uncomment line seems like mental load we could avoid.

you could also have `--snapshot` and `--release` and make it required to explicitly use one or the other.

Reply:
I'll make it default to snapshot, but running the release script will automatically handle the release versioning